### PR TITLE
Feature/boilerplate

### PR DESCRIPTION
**Pull request by aether7**
Added new boilerplate generator

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,18 +60,18 @@ Then we need to install node in our PC, if you have done that already, please ta
...
 $ node --version
-$ v5.3.0
+$ v8.2.1
 ```
 
 This should appear in response to your command, the version number might be a little different, but that is fine.
 The next step is to create all the environment we will use, so to prepare it, please download this script to create all the necessary dependencies: [LINK][react-essential-setup-source] (this script has to be run within unix terminal like OSX or linux)
 
-**NOTE** This script only works within Linux and OSX environments, if you need this script can be run under windows, please download GIT for windows from https://git-scm.com/downloads . This will create a mini "unix" environment to work with GIT and some unix commands
+**NOTE** This script only works within Linux and OSX environments
```
Comment (Mentalchaos) [inline README.md]: This tutorial isn't gonna have support for window environments now?
Just asking

File: README.md [hunk 2]
```diff
@@ -60,18 +60,18 @@ Then we need to install node in our PC, if you have done that already, please ta
 
 ```shell
 $ node --version
-$ v5.3.0
+$ v8.2.1
 ```
 
 This should appear in response to your command, the version number might be a little different, but that is fine.
 The next step is to create all the environment we will use, so to prepare it, please download this script to create all the necessary dependencies: [LINK][react-essential-setup-source] (this script has to be run within unix terminal like OSX or linux)
```
Comment (Mentalchaos) [inline README.md]: In line 67 (in parenthesis) and in line 69 (as a **NOTE**) you are saying that the script only works on osx or linux terminals
Maybe you should keep only 1 of the advises
I just read 2 times the same thing and it gets kinda repetitive for me

File: README.md (hunk 1 above)
Comment (aether7) [reply README.md]: Well, It's supposed to have every OS available to use this tutorial and boilerplate generator

File: README.md (hunk 2 above)
Comment (aether7) [reply README.md]: Ok, agreed, my bad

**Pull request merged**